### ISSUE TITLE
Editing with Synchronised Clips Causes Random Glitches During Playback

### ISSUE THREAD

**Issue by Droffelt** [latenitefilms/BRAWToolbox#116]
Hi there,
I am doing my first project with BrawTool in FCPX and i have a huge problem. When i trim a clip on the timeline the trimviewer don't move and it doesn't trim either. Strangely enough it trims when i do it at the end of the clip. See screenVid of the problem.
I hope this is some weird glitch which can be solve soon or i have to revert back to resolve to edit. Please help.
I restarted fcpx several times and also my mac studio. No solution. 

https://user-images.githubusercontent.com/131379486/233406636-24fc1fd5-d93e-446e-ae2b-841cbb35c903.mov

**Comment by latenitefilms** [comment]
Can you please try creating Multicam Clips from BRAW Toolbox instead and see if the problem remains?

https://brawtoolbox.io/faq/#i-dont-like-synchronised-clips-can-i-import-a-multicam-clips-instead

**Comment by Droffelt** [comment]
> Can you please try creating Multicam Clips from BRAW Toolbox instead and see if the problem remains?
> 
> https://brawtoolbox.io/faq/#i-dont-like-synchronised-clips-can-i-import-a-multicam-clips-instead

Hi thanks for answering. Multicam clips seems to work, but i already imported 7 days of shooting and meta tagged all shots. Costed me a day's work. Can i convert existing clips to multicam or is there a way to duplicate all metadata to the clips if i have to import again?

**Comment by latenitefilms** [comment]
Could you send me the Final Cut Pro library and I'll have a look?

You can upload here:

https://latenitefilms.digitalpigeon.com

I'll also report this issue to Apple. Are you on the latest Ventura by any chance?

One potential workaround is to "Open Clip" on each Synchronised Clip and try either move it above the primary storyline or make it a Compound Clip.

**Comment by Droffelt** [comment]
I am on Ventura 13.1 Should i update to 13.3.1 and try again?

**Comment by latenitefilms** [comment]
No, I don't think updating to Ventura 13.3.1 will have any impact on this particular problem, and if you have a lot of Audio Units, 13.3.1 could [cause issues with validating audio plugins](https://larryjordan.com/articles/if-you-havent-upgraded-to-macos-13-3-dont/).

Thanks for sending across your library. Are you able to `.ZIP` it up first, then send it across again? I think because it uploaded the library as if it was a folder, some of the file names went a bit funky. Just right click on the library and press the "Compress" option in Finder to make it a `.ZIP` file.

As far as I can tell, it seems to be a bug/problem with Final Cut Pro itself, rather than anything specifically wrong with how BRAW Toolbox works. Synchronised Clips don't seem to update/work properly when they only have a Generator inside. The only way to force Final Cut Pro to update them correctly seems to be to wrap them either in a Compound Clip or a Multicam Clip.

I will do some investigating to see if we can add something to the Workflow Extension to wrap any BRAW Toolbox Synchronised Clips inside a Multicam or Compound Clip. I'll also investigate just making **Create Multicam Clips** the default option. I'll also keep chatting with the Final Cut Pro team to see if they can provide any insight on their end.

My sincerest apologies - I know this is extremely frustrating when you've already logged 7 days of shooting!

**Comment by mdonnellymedia** [comment]
Hi @latenitefilms any progress on this issue?

I'm experiencing the exact issue mentioned in this ticket. We imported 35 days of footage without using create Multicam Clips. Bit hard to go back and do that work again. Any help would be greatly appreciated!

M.

**Comment by latenitefilms** [comment]
Have you already started editing in a project/timeline, or are you just key-wording/favouriting in the browser at this stage?

The only manual fix I know currently is to open up each Synchronised Clip, and turn the Generator into a Compound Clip. If you head down this route, obviously try it on one clip first to make sure it works before you do to lots of clips.

This is a tricky one, as I haven't been able to reproduce the issue myself. I'm on macOS Monterey 12.6.3 & FCPX 10.6.5. If you're able to create a small library that demonstrates this issue that you can send, that would be a big help.

I'm still looking into a workaround, where you can convert Synchronised Clips into Multicam Clips, but it's somewhat complex - hence why there's no quick/easy fix.

**Comment by mdonnellymedia** [comment]
> Have you already started editing in a project/timeline, or are you just key-wording/favouriting in the browser at this stage?
> 
> The only manual fix I know currently is to open up each Synchronised Clip, and turn the Generator into a Compound Clip. If you head down this route, obviously try it on one clip first to make sure it works before you do to lots of clips.
> 
> This is a tricky one, as I haven't been able to reproduce the issue myself. I'm on macOS Monterey 12.6.3 & FCPX 10.6.5. If you're able to create a small library that demonstrates this issue that you can send, that would be a big help.
> 
> I'm still looking into a workaround, where you can convert Synchronised Clips into Multicam Clips, but it's somewhat complex - hence why there's no quick/easy fix.

Thanks for the info, at the moment the issue happens randomly so were not sure how bad those fixes will stack up. We're Melbourne based and you're more than welcome to remote into our editing machine to take a look if that helps everyone else too.

**Comment by latenitefilms** [comment]
Are you able to send me your Final Cut Pro library? You can upload here:

https://latenitefilms.digitalpigeon.com

**Comment by Sanvid779** [comment]
Tengo problemas al editar, al cortar el clip,  del lugar donde corto pone el inicio del clip!! que estoy haciendo mal? 

https://user-images.githubusercontent.com/132857721/236708655-673e1ea6-8f99-4a72-bf6e-24092657ec20.mp4

**Comment by latenitefilms** [comment]
@Sanvid779 writes:

> I have problems when editing, when cutting the clip, from the place where short puts the beginning of the clip!! what am I doing wrong?

Can you please try using Multicam Clips instead?

See: https://brawtoolbox.io/faq/#i-dont-like-synchronised-clips-can-i-import-a-multicam-clips-instead

**Comment by Sanvid779** [comment]
lo intentare, la pregunta seria estos clips están grabados con audio de cámara de referencia, y con sonido directo, al hacer la multicámara como sincronizo el audio directo?

**Comment by latenitefilms** [comment]
@Sanvid779 writes:

> I'll try, the serious question are these clips recorded with reference camera audio, and with direct sound, when doing the multicam, how do I synchronize the direct audio?

If you're just using the audio on the BRAW files as a guide track, but have timecode synced your camera and an external sound recorder, I would leave the "Create Multicam Clips" off, and then use Sync-N-Link to create new Multicam Clips from the Synchronised Clips and location audio:

https://brawtoolbox.io/faq/#does-braw-toolbox-work-with-sync-n-link

You should also be able to do it manually in Final Cut Pro if you're manually syncing with a dumb slate, etc.

Basically, Synchronised Clips currently seem broken for some users in Final Cut Pro - but as soon as you "wrap" them in a Multicam Clip, everything works correctly again.

**Comment by latenitefilms** [comment]
I'm going to close this issue in favour of #128, however feel free to keep replying here if necessary.
